### PR TITLE
Ckv k8 s 102

### PR DESCRIPTION
**Pull request by niradler**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/kubernetes/checks/ApiServerEtcdCaFile.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.kubernetes.base_spec_check import BaseK8Check
+
+
+class ApiServerEtcdCaFile(BaseK8Check):
+    def __init__(self):
+        # CIS-1.6 1.2.32
```
Comment (nimrodkor) [inline checkov/kubernetes/checks/ApiServerEtcdCaFile.py:8]: @niradler Please remove this line

File: checkov/kubernetes/checks/ApiServerEncryptionProviders.py [hunk 2]
```diff
@@ -28,9 +28,9 @@ def scan_spec_conf(self, conf):
                 else:
                     keys.append(cmd)
                     values.append(None)
-
-        if "--encryption-provider-config" not in keys:
-                    return CheckResult.FAILED
+        if "kube-apiserver" in keys:
+            if "--encryption-provider-config" not in keys:
+                        return CheckResult.FAILED
```
Comment (nimrodkor) [inline checkov/kubernetes/checks/ApiServerEncryptionProviders.py:33]: @niradler On purpose?

**Review by nimrodkor — COMMENTED** [review]
@niradler Nice

File: checkov/kubernetes/checks/ApiServerEncryptionProviders.py (hunk 2 above)
Comment (niradler) [reply checkov/kubernetes/checks/ApiServerEncryptionProviders.py:33]: yes

File: checkov/kubernetes/checks/ApiServerEtcdCaFile.py (hunk 1 above)
Comment (niradler) [reply checkov/kubernetes/checks/ApiServerEtcdCaFile.py]: we dont need ?  # CIS-1.6 1.2.32

**Review by nimrodkor — APPROVED** [review]
Nice!

**Pull request merged**